pkg/controller/controller.go

```diff
@@ -74,7 +74,12 @@ func RegisterInformers(c *config.Config) {
 			if !ok {
 				continue
 			}
-			for _, ns := range r.Namespaces {
+
+			// Populate namespaces from the config file for adding resource informers
+			r.Namespaces.Include = utils.PopulateNamespacesForResources(r.Namespaces.Include, r.Namespaces.Ignore)
```

Comment:
We should not mutate `r.Namespaces.Include` from users input. You can have temp variable if required
OR directly use `PopulateNamespacesForResources` call in range

Reply:
resloved it